main.go

```diff
@@ -241,6 +242,13 @@ func parseDefaultConfigs(parser *flags.Parser) error {
 }
 
 func runWithOptions(opts Options, out output) {
+	if opts.TOpts.HostPortFile == "?" {
```

Comment:
we can unit test this in, `main_test.go` has some integration tests that should have examples of how to do this.